### PR TITLE
Add length to workshop

### PR DESCRIPTION
**Pull request by jabawack81**
As Per issue #1104 this PR add the length field to workshops

### CODE REVIEW

File: app/views/workshops/show.html.haml [hunk 1]
```diff
@@ -4,7 +4,12 @@
       %h2
         Workshop at #{@workshop.host.name}
         %br
-        %small #{humanize_date(@workshop.date_and_time, with_time: true)} #{@workshop.distance_of_time}
+        %small #{humanize_date(@workshop.date_and_time)} #{@workshop.distance_of_time}
+        #workshop-time.time
+          %i.material-icons
+            access_time
+          = @workshop.time
+          = @workshop.ends_at.present? ? l(@workshop.ends_at, format: :time) : ""
```
Comment (notapatch) [inline app/views/workshops/show.html.haml]: Missing hyphen between start and end time - this is what I'm seeing 

---
Workshop show

**Comment by notapatch** [comment]
You don't have an end time on the event index but reading through the issue and I couldn't decide if they wanted the end time on the events index or not. They stated: 

> Pages that are missing the workshop length:
>      https://codebar.io/events

then said:

> We don't necessarily need to add to all pages, the workshop page should be enough

*scratches head* 

If you're happy. I'm happy.

---
I should state: I am _not_ an official reviewer. 

Regardless to everything, I've played around with the pull request. It worked for me. The code looks fine. Good luck!

**Comment by notapatch** [comment]
When you've fixed the workshop/show page ... are you OK with me plastering the screenshots in the issue? Probably best for the users to see something straight away? Easier to change now than in a weeks time...

File: app/views/workshops/show.html.haml (hunk 1 above)
Comment (jabawack81) [reply app/views/workshops/show.html.haml]: thank you I missed that, now is fixed

**Comment by jabawack81** [comment]
@notapatch thanks for the inputs

I've fixed the two problems you found and organized the commits.

In regard to the validation of the `end_at` I was wondering about that too, at the moment is not validated in events so I skipped that for the moment and looking at the DB `ends_at` is saved with the date of the day when the event/workshop is edited, I konw this is not a big deal because the app is using only the time from it but I'm planning to add a before save hook in events and workshop to use the begins at date in the `ends_to` and then add a validation in both.

File: app/views/admin/workshops/_edit_form.html.haml [hunk 2]
```diff
@@ -8,6 +8,8 @@
   .row
     .large-3.columns
       = f.input :local_time, as: :string, required: true, input_html: { data: { value: @workshop.time.try(:strftime, '%H:%M') } }
```
[the remark this answers is not in the document]
Comment (matyikriszta) [reply app/views/admin/workshops/_edit_form.html.haml]: @jabawack81 I agree, can we change this to Begins at please?

**Comment by jabawack81** [comment]
@matyikriszta done

**Comment by matyikriszta** [comment]
@jabawack81 would you mind doing a quick rebase on this?

**Comment by jabawack81** [comment]
sorry messed up clicking on the wrong button

**Comment by matyikriszta** [comment]
> sorry messed up clicking on the wrong button

@jabawack81 no worries, all good

**Comment by KimberleyCook** [comment]
🎉🎉🎉🎉🎉🎉

**Pull request merged**